tests/features/solution/add_solution.feature

```diff
@@ -247,3 +247,35 @@ Feature: "Add solution" visibility options.
     Then I delete the "Climate change tracker" solution
     And I delete the "Climate change tracker" solution
     And I delete the "Angela Crespi" contact information
+
+  Scenario: Check solution downloads counter
+    # Create the dummy data to work with.
+    Given the following collection:
+      | title              | Ocean studies |
+    Given the following solution:
+      | title       | Climate change tracker |
+      | description | Atlantic salmon arrived after the Little Ice Age. |
+      | collection  | Ocean studies                                     |
+      | state       | validated                                         |
+    Given the following distributions:
+      | title        | description | parent | downloads |
+      | Sample distribution | Sample description | Climate change tracker | 10 |
+      | Sample distribution 2 | Sample description 2 | Climate change tracker | 0 |
+    # Checks solutions tiles to we have the correct numbers.
+    Then I visit "/solutions"
+    Then I should see the text "10"
+    When I go to the homepage of the "Climate change tracker" solution
+    # Check the same thing on the homepage of the solutions.
+    Then I should see the text "Downloads: 10"
+    When I delete the "Sample distribution" asset distribution
+    Given the cache has been cleared
+    And I reload the page
+    # Check it again after the first distribution is deleted and cache has been cleared.
+    Then I should not see the text "Downloads: 10"
+    # Go back to the solution tiles page and check downloads are disappeared.
+    Then I visit "/solutions"
+    Then I should not see a "icon--download" element
+    # Delete all created dummy data.
+    When I delete the "Sample distribution 2" asset distribution
+    And I delete the "Climate change tracker" solution
+    And I delete the "Ocean studies" collection
```

Comment:
You don't need the last 3 lines. Behat automatically cleans up the entities created via API. Only if you create via UI, you'll need to take care.

Reply:
Removed!